pymove/_version.py

```diff
@@ -391,8 +354,7 @@ def render_pep440_pre(pieces):
 
 
 def render_pep440_post(pieces):
-    """
-    TAG[.postDISTANCE[.dev0]+gHEX] .
+    """TAG[.postDISTANCE[.dev0]+gHEX] .
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)